### PR TITLE
Password restrictions not accepting valid password Fixed

### LINKED ISSUE #51: Password restrictions not accepting valid password
Code : https://gist.github.com/dcc4157b058a560a5fa6545816fe0b5d

image

### PR DESCRIPTION
**Pull request by kdsinghcoder**
Fixed #51

### CODE REVIEW

File: views/account/register.hbs [hunk 1]
```diff
@@ -50,7 +50,7 @@
                     </div>
                     <div class="p-1">
                         <input type="password" name="password" id="password" placeholder="Password"
-                            pattern="^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[@$!%*?&])[A-Za-z\d@$!%*?&]{8,}$"
+                            pattern="^(?=.*?[A-Z])(?=.*?[a-z])(?=.*?[0-9])(?=.*?[#?!@$%^&*-]).{8,}$"
```
Comment (YashKumarVerma) [inline views/account/register.hbs:53]: image

Still not accepting this. "." is a special character.

**Review by YashKumarVerma — CHANGES_REQUESTED** [review]

**Review by kdsinghcoder — COMMENTED** [review]
fixed issue #52 

will now accept "." also

**Review by YashKumarVerma — CHANGES_REQUESTED** [review]
Too many changes / commits

**Comment by YashKumarVerma** [comment]
Closing due to inactivity.

**Pull request closed without merging**